samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -96,6 +98,8 @@ The following table shows the mesh light fixture composition data for this sampl
    +-------------------------------+----------------------------+
    | Sensor Setup Server           |                            |
    +-------------------------------+----------------------------+
+   | Large Comp Data Server        |                            |
```

Comment:
I thought you were just using `light_ctrl` for your own testing purpose. I'm really uncertain about inserting LCD Server in this sample. This sample is our "the BLC NLC and ENM NLC sample". None of those specs mandate usage of LCD server. So, adding it here will simply bloat the sample and make this sample a tad bit more complicated.

I would highly recommend to not add metadata into this sample.

Instead of this, could we rather add these metadata in LCD test, mesh_shell, and in PTS tester, like done for health server? These things could be done as a seperate task.

Reply:
Changed light_ctrl sample, because that was part of the DoD in the task :) but we can discuss this when I am in the office after lunch more? Will update a cleaned-up version of this PR after.

Reply:
No idea. @omkar3141 initially created the task. Seems he disagrees at the moment to what he thought about this in time of task creation. :)
I'm Ok to not add it in sample either. I'm not sure if we have to add metadata artificially to anywhere at all. If we need to check whether it works or doesn't just use unit testing.

Reply:
and @omkar3141 if you look at the [task description](https://nordicsemi.atlassian.net/browse/NCSDK-22500), one goal was to add it to a sample, not just mesh_shell.